shell/platform/fuchsia/flutter/vsync_recorder.cc

```diff
@@ -27,25 +27,24 @@ VsyncRecorder& VsyncRecorder::GetInstance() {
 VsyncInfo VsyncRecorder::GetCurrentVsyncInfo() const {
   {
     std::unique_lock<std::mutex> lock(g_mutex);
-    if (last_presentation_info_) {
-      return {fml::TimePoint::FromEpochDelta(fml::TimeDelta::FromNanoseconds(
-                  last_presentation_info_->presentation_time)),
-              fml::TimeDelta::FromNanoseconds(
-                  last_presentation_info_->presentation_interval)};
-    }
+    return {fml::TimePoint::FromEpochDelta(fml::TimeDelta::FromNanoseconds(
```

Comment:
This seems to be different behavior from what exists today. Seems like a regression in terms of feature parity between `Present` and `Present2`. Can you briefly describe the difference in behavior users of flutter can expect by switching away from `presentation_interval` to a constant `1/60`?